server/repository/src/mock/mod.rs

```diff
@@ -340,6 +340,7 @@ impl MockDataInserts {
     }
 
     pub fn stores(mut self) -> Self {
+        self.names = true;
```

Comment:
I've done this in other projects, I can't see any harm in adding these dependancies in?

Reply:
makes sense to have them in, thanks!